streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamSessionWindowAggregate.java

```diff
@@ -81,6 +81,7 @@ public void enableSendingOldValues() {
         private StreamsMetricsImpl metrics;
         private InternalProcessorContext internalProcessorContext;
         private Sensor lateRecordDropSensor;
+        private long observedStreamTime = -1L;
```

Comment:
nit: use `ConsumerRecord.NO_TIMESTAMP` in stead of `-1L` (same below)

Reply:
Hmm, should we use Long.MIN_VALUE instead, to account for negative timestamps?

We don't need a sentinel value, we just need something that's not larger than a real record's timestamp.

Reply:
So far, we don't support negative timestamp, so it seems to be cleanest to use `NO_TIMESTAMP` for now -- if we ever introduce negative timestamps, we need to update the code anyway.

Reply:
Sounds good. It's updated.